src/include/kdbproposal.h

```diff
@@ -53,6 +53,10 @@ enum elektraLookupOptions
 	KDB_O_NOSPEC = 1 << 18,      ///< Do not use specification for cascading keys (internal)
 	KDB_O_NODEFAULT = 1 << 19,   ///< Do not honor the default spec (internal)
 	KDB_O_CALLBACK = 1 << 20     ///< For spec/ lookups that traverse deeper into hierarchy (callback in ksLookup())
+#ifdef ELEKTRA_ENABLE_OPTIMIZATIONS
```

Comment:
This can be problematic: it means that code cannot be built if ELEKTRA_ENABLE_OPTIMIZATIONS is not set.